code/ui/lisibilite_ui_app/views.py

```diff
@@ -0,0 +1,119 @@
+from django.shortcuts import render, redirect
+from django.views import View
+from lisibilite.Lisibilite import Lisibilite
+
+
+class HomePageView(View):
+
+	def get(self, request):
+		"""
+		The function to handle the HTTP GET request for home page.
+		:param request: The django HTTP request object
+		:return The render object
+		"""
+		return render(request, 'index.html')
+
+	def post(self, request):
```

Comment:
If post is rendering none, do we need it? Do you see a future case?

Reply:
Not really. Added it as a part of the code skeleton in the beginning. Will remove it.

Reply:
I have pushed the changes related to this.